src/utils/mtev_lru.c

```diff
@@ -0,0 +1,252 @@
+#include "mtev_lru.h"
+#include "mtev_hash.h"
+#include "mtev_log.h"
+
+#include <ck_hs.h>
+#include <sys/queue.h>
```

Comment:
I _think_ sys/queue.h exists on all the platforms we care about (not sure about Windows?), but it makes me nervous to use it, since it's not POSIX.